technical/validations.md

```diff
@@ -7,20 +7,35 @@ these validations will be duplicated in the front-end code for an optimal user e
 ## Entity-level validations
 
 ### Playground
-- The name of the playground is between 3 en 40 characters
-- Two playgrounds cannot be located within .. metres of each other
+- The name of the playground is between 3 en 40 characters ( no whitespaces in the beginning and trailing of the playground name )
+- Two playgrounds cannot be located within .. metres of each other ( Are there any API/Service provided by Netherlands to verify entered/chosen address )
 - The maximum number of playgrounds that can exist within the system is ..
+- At least 4(TBD) decimal precisions required for Longitude and Latitude of the Playground location, in order to identify how close two Playgrounds are located
+- A User must not be able to join a Playground twice
```

Comment:
First be more strict: I think you mean that a user cannot join a playground he is already a volunteer in. He should be able to rejoin a playground that he left before.
Second: Currently we just ignore this command if the user is already in the playground and return success. Do you . want to change this behaviour? Please document here what you intend to do.